<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messenger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ContactsCountLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Total: </string>
    </property>
   </widget>
   <widget class="QListWidget" name="contactsListWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>261</width>
      <height>361</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteContactPushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ChatBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>641</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addContactPushButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>70</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Self</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="chatTextLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Chat With:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="delContactLineEdit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>contact or chat room to del</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="messageSendTextEdit">
       <property name="placeholderText">
        <string>type message here </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendMessagePushButton">
       <property name="text">
        <string>Send Message</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
